pyscriptjs/tests/integration/conftest.py

```diff
@@ -130,6 +130,13 @@ class HTTPServer(SuperHTTPServer):
     Keyboard Interrupt passes.
     """
 
+    def __init__(self, host_and_port, handler_class):
+        self.handler_class = handler_class
+        super().__init__(host_and_port, handler_class)
+
+    def disable_cors(self):
+        self.handler_class.use_cors = False
```

Comment:
I would call this method `disable_cors_headers()`, it seems cleaner

Reply:
done